### PR TITLE
for portainer add-on

### PR DESCRIPTION
**Pull request by balasu**

### CODE REVIEW

File: tests/test-addons.py [hunk 1]
```diff
@@ -324,6 +325,16 @@ def test_multus(self):
...
+    
+    def test_portainer(self):
+        """
+        Sets up and validates Portainer.
+        """
+        print("Enabling Portainer")
+        microk8s_enable("portainer")
+        print("Validating Portainer")
+        validate_portainer()
+        print("Disabling Portainer")
```
Comment (ktsakalozos) [inline tests/test-addons.py:337]: I think you are missing the disable command here.

**Review by ktsakalozos — CHANGES_REQUESTED** [review]
Thank you for this PR @balasu . You will also need to add your addon at https://github.com/ubuntu/microk8s/blob/master/microk8s-resources/wrappers/addon-lists.yaml

**Review by balchua — COMMENTED** [review]
We may want to keep `$ARCH` here `cdkbot/microbot-amd64`, i think the team ran the tests on multiple arch too.

**Review by ktsakalozos — APPROVED** [review]
LGTM +1 thanky ou for the PR @balasu

**Pull request merged**